src/Platforms/OraclePlatform.php

```
@@ -431,13 +431,43 @@ protected function _getCreateTableSQL($name, array $columns, array $options = []
         return $sql;
     }
 
+    public function getListDatabaseIndexesSQL(string $database): string
+    {
+        return $this->getListIndexesSQL($database);
+    }
+
     /**
      * {@inheritDoc}
      *
      * @link http://ezcomponents.org/docs/api/trunk/DatabaseSchema/ezcDbSchemaOracleReader.html
      */
     public function getListTableIndexesSQL($table, $database = null)
     {
+        return $this->getListIndexesSQL($database ?? '', $table);
```

I'm not sure if we should silently replace null with an empty string. An empty string isn't a valid database name.

Should be relevant to https://github.com/doctrine/dbal/issues/3826. I believe I have some work in progress stashed.

---

fixed